src/netstandard/ref/System.Runtime.CompilerServices.cs

```diff
@@ -10,6 +10,20 @@ public sealed partial class AccessedThroughPropertyAttribute : System.Attribute
         public AccessedThroughPropertyAttribute(string propertyName) { }
         public string PropertyName { get { throw null; } }
     }
+    public partial struct AsyncIteratorMethodBuilder
+    {
+        private object _dummy;
```

Comment:
should private types be included here?

Reply:
It's necessary for the C# compiler to not see this as an empty struct.

Reply:
Indeed. See #1012 for details.